services/QuillLMS/db/big_query/views/reporting_sessions_view.sql

```diff
@@ -0,0 +1,38 @@
+SELECT
+ activity_sessions.user_id as student_id,
+ activity_sessions.activity_id as activity_id,
+ activities.name as activity_name,
+ schools.id as school_id,
+ schools.name as school_name,
+ classrooms_teachers.user_id as teacher_id,
+ users.name as teacher_name,
+ classrooms.id as classroom_id,
+ classrooms.grade as grade,
+ DATETIME_TRUNC(activity_sessions.completed_at, DAY) as completed_date,
```

Comment:
I believe this makes it slightly more efficient to query by the simpler date, but I've added so much grouping to the query since I originally added this so it might not have that much value (so it could be a take or leave). There is something nice about making the column a simple date since our time filters will always be a day granularity. So this is a reasonable shape.

Reply:
Will leave this for now, but can edit/change this in the PR to use this code.